docs/tutorial/grid_comps/CMakeLists.txt

```diff
@@ -4,4 +4,4 @@ add_subdirectory (leaf_comp_a)
 add_subdirectory (leaf_comp_b)
 add_subdirectory (parent_with_one_child)
 add_subdirectory (parent_with_two_children)
-add_subdirectory (code_generator)
+add_subdirectory (automatic_code_generator)
```

Comment:
```suggestion
add_subdirectory (automatic_code_generator_example)
```

Reply:
@tclune I assume that you change the name from `automatic_code_generator` to `automatic_code_generator_example`. The documentation file also needs to be edited to reflect the name of the directory.

Reply:
I did not change the name.   I made a suggestion in GitHub for the change.   You have apparently not committed that suggestion.  (Possibly you manually did something for that instead.)   But in either event, yes, I expected you to ensure it is all consistent.

Reply:
Changed the directory name and pushed the changes to the branch.